### DISCUSSION TITLE
Show the same window in multiple tabs

### DISCUSSION THREAD

**Discussion by TonPott** [kovidgoyal/kitty#6543] [Q&A] [answered]
Hi,
I wonder if it its possible to show the same window in different tabs. Like, for example I need a log or text file open as reference while working in different windows, but when I have all of them on one screen at the same time, it gets cluttered. So I want multiple tabs but in all of them I see the content of the file I'm referencing.
Of course I could just open it in multiple windows, but a clone of the same shell would be nice.

**Comment by kovidgoyal** [comment]
You can clone a shell with clone-in-kitty
but you will still need to re-open the file.
https://sw.kovidgoyal.net/kitty/shell-integration/#clone-shell

**Reply by TonPott** [reply]
Yes I saw that, but then things like scrolling etc are not shared. I was thinking more of a second representation of the same window in different tabs.
But it seems that's not possible (yet?).
Thanks anyway for the quick reply

**Comment by kovidgoyal** [comment, marked as answer]
No, it's not.
